docs/github-pages-example-3.html

```diff
@@ -0,0 +1,17 @@
+<div>
+    <div id="wrapper-3">
+    </div>
+
+    <script type="text/javascript">
+        dtCreateDataTable({
+            wrapperDivId: 'wrapper-3',
+            numRows: 4,
+            numColumns: 3,
+            rowsNamePlural: 'favorite cartoon characters',
+            columnsName: 'Round',
+            rowsName: 'Favorite cartoon character',
+            names: ["# Votes", "Status"],
+            values: [0, ["Active", "Eliminated", "Elected"]],
```

Comment:
Why does a favorite cartoon character have "Active/Eliminated/Elected" and "# Votes/Status"?

Is this trying to simulate a ranked-choice voting election, except with the specific use case of a Favorite Cartoon Character poll? If so, I don't think that's sufficiently clear here - if you want an RCV example, can you clarify that somehow?

Also: this example isn't actually being used anywhere- notice how index.md did not change. You defined what `{{ex3}}` is, but never actually used `{{ex3}}` - so this PR has no visible effects